pkg/sql/logictest/testdata/logic_test/fk

```diff
@@ -1349,3 +1349,122 @@ ALTER TABLE t ADD FOREIGN KEY (a, b) REFERENCES t (a, b)
 
 statement ok
 DROP TABLE t
+
+subtest 26748
+
+statement ok
+CREATE TABLE a (
+  x STRING NULL
+ ,y STRING NULL
+ ,CONSTRAINT "primary" PRIMARY KEY (y, x)
+);
+
+statement ok
+CREATE TABLE b (
+ a_y STRING NULL
+ ,a_x STRING NULL
+ ,CONSTRAINT fk_ref FOREIGN KEY (a_y, a_x) REFERENCES a (y, x)
+);
+
+statement ok
+INSERT INTO a (x, y) VALUES ('x1', 'y1')
+
+statement error pq: missing value for column "a_y" in multi-part foreign key
+INSERT INTO b (a_x) VALUES ('x1')
+
+statement error pq: missing value for column "a_x" in multi-part foreign key
+INSERT INTO b (a_y) VALUES ('y1')
+
+statement error pq: foreign key violation: value \['y1' NULL\] not found in a@primary \[y x\]
+INSERT INTO b (a_y, a_x) VALUES ('y1', NULL)
+
+statement error pq: foreign key violation: value \[NULL 'x1'\] not found in a@primary \[y x\]
+INSERT INTO b (a_y, a_x) VALUES (NULL, 'x1')
+
+statement ok
+INSERT INTO b (a_x, a_y) VALUES ('x1', 'y1')
+
+statement ok
+INSERT INTO b (a_x, a_y) VALUES (NULL, NULL)
```

Comment:
whoa TIL

Reply:
Yep.  It's a non-referencing value.

Sadly, we don't support MATCH FULL properly.  See #30026.